homeassistant/components/electric_kiwi/__init__.py

```diff
@@ -0,0 +1,71 @@
+"""The Electric Kiwi integration."""
+from __future__ import annotations
+
+import aiohttp
+from electrickiwi_api import ElectricKiwiApi
+from electrickiwi_api.exceptions import AuthException
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryAuthFailed, ConfigEntryNotReady
+from homeassistant.helpers import aiohttp_client, config_entry_oauth2_flow
+
+from . import api
+from .const import DOMAIN
+from .coordinator import ElectricKiwiHOPDataCoordinator
+
+PLATFORMS: list[Platform] = [
+    Platform.SENSOR,
+    Platform.SELECT,
```

Comment:
Please reduce this PR to a single platform, as per:

<https://developers.home-assistant.io/docs/creating_component_code_review#5-make-your-pull-request-as-small-as-possible>

And:

<https://developers.home-assistant.io/docs/review-process>

I would recommend keeping the sensor platform and remove the select platform for now.